src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -866,16 +866,27 @@ void SpatialPooler::inhibitColumnsLocal_(const vector<Real> &overlaps,
 
 
       if (wrapAround_) {
+        numNeighbors = 0;  // In wrapAround, number of neighbors to be cons     idered is solely a function of the inhibition radius, the number of dimensi     ons, and of the size of each of those dimenion
+         UInt predN = 1;
+         for (UInt i = 0; i<columnDimensions_.size();i++) {
+           UInt diam = 2*inhibitionRadius_ + 1;
+           predN *= ( diam + ((columnDimensions_[i] - diam) * ((UInt) (columnDimensions_[i] - diam) >> (shftInt))) ); // max(diam, columDimenions_[i]);
+         }
+         predN -= 1;
+         numNeighbors = predN;
+         const UInt numActive_wrap = (UInt)(0.5f + (density * (numNeighbors + 1)));
```

Comment:
this one should round, or crop?

Reply:
What do you mean, round or crop?  Are talking specifically about the "const UInt numActive_wrap" line? I copied that from the code without modification. Or was it something else?

Reply:
yes, about that line. Ok if you've just copied that. I wanted to be clear if the cropping (cast to integer type) is intended and ok.

Reply:
Yes, definitely copied. The copied line is still in line 906 of the pull-request file (and here's a link to that same line on the current htm.core master branch: https://github.com/htm-community/htm.core/blob/fabf17676203ec6c06d23d0f2de7d9316c93b74e/src/htm/algorithms/SpatialPooler.cpp#L895).